lib/api/attributes.js

```diff
@@ -1,10 +1,12 @@
 var _ = require('lodash'),
+  static = require('../static'),
```

Comment:
`static` is [reserved][1] as a future keyword. Perhaps we should use `static_` instead.


[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Future_reserved_keywords